sources/packages/web/src/components/students/modals/PDStatusApplicationModal.vue

```diff
@@ -12,10 +12,12 @@
       </p>
     </template>
     <template v-slot:footer>
-      <v-btn variant="outlined" color="primary" @click="requestPD(false)"
-        >No</v-btn
-      >
-      <v-btn color="primary" @click="requestPD(true)">Yes</v-btn>
+      <v-row class="m-0 p-0">
```

Comment:
Since the margin and padding are 0 is there any need for this class ?

Reply:
m and p and not initially 0 here and it fixes the miss alignment.